docs/maintainer/knowledge_base.md

```diff
@@ -1271,6 +1271,40 @@ which has `openblas` as a dependency and has a symlink from `libblas.so.3` to `l
 BLAS `3.8.0` API.  This means that, at install time, the user can select what BLAS implementation
 they like without any knowledge of the version of the BLAS implementation needed.
 
+### Microarchitecture-optimized builds {#microarch}
+
+conda [virtual packages](../glossary.md#virtual-package) include `__archspec`, which expose the processor architecture to the solver. However, `__archspec` should not be used directly in recipes; instead, users should rely on the [`microarch-level`](https://github.com/conda-forge/microarch-level-feedstock) helper packages (contributed in [staged-recipes#24306](https://github.com/conda-forge/staged-recipes/pull/24306)).
+
+Before learning how to use it, please read these considerations:
+
+- Adding microarchitecture variants can result in too many entries in the build matrix. Do not overuse it.
+- These optimized builds should only be used when the performance improvements are significant.
+- Preferrably, the project should rely on runtime dispatch for arch-specific optimizations.
+- If the package is already too large, consider using smaller outputs for the arch-optimized variants.
+
+To implement microarchitecture-optimized builds in your feedstock, you'll end up with something like:
+
+```yaml title="recipe/conda_build_config.yaml"
+microarch_level:  # [unix and x86_64]
+  - 1  # [unix and x86_64]
+  - 3  # [unix and x86_64]
+  - 4  # [unix and x86_64]
+```
```

Comment:
Actually, the very recent [archspec 0.2.3](https://github.com/archspec/archspec/releases) now has windows support, in large part due to @isuruf's work on this. 🥳 

Not sure what else is necessary to wire this up though, just tried on a fully up-to-date environment:
```
>conda info
    [...]
       virtual packages : __archspec=1=x86_64
                          __conda=24.1.2=0
                          __win=0=0
```